src/main/java/com/qaprosoft/carina/demo/gui/pages/AutomationPracticeSearchResultPage.java

```diff
@@ -0,0 +1,35 @@
+package com.qaprosoft.carina.demo.gui.pages;
+
+import com.qaprosoft.carina.core.foundation.webdriver.decorator.ExtendedWebElement;
+import com.qaprosoft.carina.core.gui.AbstractPage;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.FindBy;
+
+public class AutomationPracticeSearchResultPage extends AbstractPage {
```

Comment:
Could you attach screenshot what it is page?

Reply:
<img width="1680" alt="Screen Shot 2022-04-18 at 13 22 56" src="https://user-images.githubusercontent.com/103547507/163839267-efd0eec4-f3b1-493b-a7d7-45051b05d263.png">

Reply:
That would be the search result page, the one where user gets redirected to after submiting the searchbox